lib/Service/RecordingService.php

```diff
@@ -218,14 +282,35 @@ public function notifyStoredRecording(Room $room, Participant $participant, File
 		$this->notificationManager->notify($notification);
 	}
 
+
+	public function notifyStoredTranscript(Room $room, Participant $participant, File $file): void {
+		$attendee = $participant->getAttendee();
+
+		$notification = $this->notificationManager->createNotification();
+
+		$notification
+			->setApp('spreed')
+			->setDateTime($this->timeFactory->getDateTime())
+			->setObject('recording', $room->getToken())
+			->setUser($attendee->getActorId())
+			->setSubject('transcript_file_stored', [
+				'objectId' => $file->getId(),
+			]);
+		$this->notificationManager->notify($notification);
+	}
+
 	public function notificationDismiss(Room $room, Participant $participant, int $timestamp): void {
 		$notification = $this->notificationManager->createNotification();
 		$notification->setApp('spreed')
 			->setObject('recording', $room->getToken())
-			->setSubject('record_file_stored')
 			->setDateTime($this->timeFactory->getDateTime('@' . $timestamp))
 			->setUser($participant->getAttendee()->getActorId());
 		$this->notificationManager->markProcessed($notification);
```

Comment:
Is this `$this->notificationManager->markProcessed($notification);` still needed? Is not enough with the new one added below?

Reply:
Right, that should be removed